web/src/engine/websites/ManHuaGui.ts

```diff
@@ -0,0 +1,49 @@
+import { Tags } from '../Tags';
+import icon from './ManHuaGui.webp';
+import { DecoratableMangaScraper } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import * as SinMH from './decorators/SinMH';
+
+/*
+this.config = {
```

Comment:
Comment can probably be removed, the information will still be available in HakuNeko legacy code